src/main/java/nl/cherement/jak/service/EventService.java

```diff
@@ -18,12 +21,21 @@ public EventService(EventRepository repository) {
         super(repository);
     }
 
-    public List<EventEntity> getByBoardId(Long boardId) {
-        return repository.findByTo_BoardId(boardId);
+    public List<EventEntity> getByBoard(BoardEntity boardEntity) {
+        return repository.findByTo_Board(boardEntity);
+    }
+
+    public void createEvent(Authentication authentication, CardEntity updatedCardEntity, CardEntity currentCardEntity) {
```

Comment:
Is het niet `save` ?

Reply:
Nee want we maken hier het event zelf aan. Bij save slaan we alleen een object op in de db die we al hebben